nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/src/test/java/org/apache/nifi/web/server/JettyServerTest.java

```diff
@@ -155,29 +155,4 @@ public void testConfigureSslContextFactoryWithPkcsTrustStore() {
         verify(contextFactory).setTrustStoreType(trustStoreType);
         verify(contextFactory).setTrustStoreProvider(BouncyCastleProvider.PROVIDER_NAME);
     }
-
-    @Test
-    public void testNoDuplicateXFrameOptions() throws NoSuchFieldException, IllegalAccessException, ServletException, IOException {
```

Comment:
Was this test moved elsewhere or just removed?

Reply:
Removed. I think this was simply a bad test and that what it was checking for is handled by another unit test in HTTPHeaderFiltersTest.

I also think I should probably add some integration tests to check for these HTTP headers in actual responses, but I didn't know where to get an example on how we do that.